src/App.css

```diff
@@ -1,3 +1,111 @@
-body {
+:root {
+  --bubble-gum-pink: #ff5ab7;
+  --black: #1e1e1e;
+  --pinkish-grey: #c7c7c7;
+  --very-light-pink: #c4c4c4;
+  --pale-grey: #f5f6fa;
+  --medium-blue: #307bbe;
+  --white: #fafafa;
+  --white-two: #fff;
+  --white-three: #e8e8e8;
+  --azure: #0290ff;
+  --black-5: rgba(0, 0, 0, 0.05);
+}
+
+.routes {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  width: 100%;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+#root {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.body {
+  display: flex;
```

Comment:
- [ ] You have used display flex many times which means you are repeating yourself and this falls under the category of repeating yourself which isn't recommended and is considered bad css practice, I would suggest you create a class with dispaly: flex inside of it and then use it whenever you need on your html 
![image](https://github.com/yashodhicy/BookStore/assets/62705947/1c89d16c-1518-405a-b113-daff02b19f2b)